docs/USAGE.md

```diff
@@ -171,6 +171,14 @@ Add --no-sandbox argument to the guest program.
 
 ## Compatibility
 
+### BOX64_AVX
+
+Expose AVX extension to CPUID and cpuinfo file.
```

Comment:
because it's dafault value depends on the architecture, maybe it should be noted in the description that it's eanbled at 2 for arm build and disable by default on other architecture?

Reply:
Indeed, I forgot that.